sources/AdvancedWorkingDirectoryWindowController.h

```diff
@@ -14,6 +14,6 @@
 @property(nonatomic, copy) Profile *profile;
 
 // An array of keys that may be changed in self.profile. All take string values.
-- (NSArray *)allKeys;
+@property (readonly, strong) NSArray<NSString*> *allKeys;
```

Comment:
Ah, you are right.
I was trying to make it more ARC and Swift friendly.

Should I revert it?